serve/README.md

```diff
@@ -202,6 +202,11 @@ Here, `code` and `message` represent the status of the request.
 probability of each class, could be the IDs of output sentence, and so
 on.
 
+## MNIST Demo Client
+
+If you have trained an model with [train.py](https://github.com/reyoung/paddle_mnist_v2_demo/blob/master/train.py) and
```

Comment:
Why this train.py is not in PaddlePaddle repo? Should we import it in this PR?

Reply:
@reyoung 
I think we need to import it to this PR